tickets/src/test/java/no/unit/nva/publication/ticket/TicketRequestTest.java

```diff
@@ -0,0 +1,20 @@
+package no.unit.nva.publication.ticket;
+
+import static org.junit.jupiter.api.Assertions.assertInstanceOf;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import no.unit.nva.commons.json.JsonUtils;
+import org.junit.Test;
+
+public class TicketRequestTest {
+
+    @Test
+    public void shouldSerializeRequestBodyWithoutTypeAsUpdateTicketRequest() throws JsonProcessingException {
+        var json = """
```

Comment:
To support "old" request body, so frontend gets some time to take in use "type" property.

Reply:
Mark as deprecated